### PR TITLE
More cleanup in poller

### PR DESCRIPTION
**Pull request by brittcyr**

### CODE REVIEW

File: src/poller.ts [hunk 1]
```diff
@@ -8,82 +8,62 @@ import {
...
+      const dipDeposit: DIPDeposit = {
+        splTokenName: this.splTokenName,
+        premiumAssetName: this.premiumAssetName,
         expirationMs: this.expirationSec * 1_000,
-        strikeUsdcPerToken: this.strike,
-        callOrPut: this.type,
-        qtyTokens: new_amount / 10 ** decimals,
+        strikeUsdcPerToken: this.strikeTokens,
+        callOrPut: this.callOrPut,
+        qtyTokens: newAmount / 10 ** NUM_DIP_ATOMS_PER_TOKEN,
```
Comment (DonDuala) [inline src/poller.ts:64]: Won't this break on SOL?

Comment (brittcyr) [reply src/poller.ts:64]: No. code wasnt doing anything. this.splTokenName is a name, not an address.

Comment (DonDuala) [reply src/poller.ts:64]: Don't we still need it corrected then? wSOL has 9 decimals.. what am I missing?

Comment (brittcyr) [reply src/poller.ts:64]: We are watching the option address which is always 6 decimals, not the vault token account

**Review by DonDuala — APPROVED** [review]

**Review by DonDuala — APPROVED** [review]

**Pull request merged**